modular_skyrat/master_files/code/modules/mob/living/emote_popup.dm

```diff
@@ -6,29 +6,62 @@
 	appearance_flags = APPEARANCE_UI_IGNORE_ALPHA | KEEP_APART
 	mouse_opacity = 0
 
-/// Flicks a emote over a mob for a determinable amount of time. Can only be used on mob/living.
+/**
+ * Flick_emote_popup_on_(???) -- A proc type that, when called, causes a image/sprite to appear above whatever entity it is called on.
```

Comment:
You don't need to add the name of the proc in its own documentation, it's redundant because it's already added by default.

Reply:
Yep, I was gonna do that but I forgot to remove all of them